frontend/src/scenes/ingestion/v2/IngestionWizard.tsx

```diff
@@ -41,8 +42,9 @@ export function IngestionWizardV2(): JSX.Element {
             {currentView === INGESTION_VIEWS.CHOOSE_FRAMEWORK && <FrameworkPanel />}
             {currentView === INGESTION_VIEWS.WEB_INSTRUCTIONS && <InstructionsPanel />}
             {currentView === INGESTION_VIEWS.VERIFICATION && <VerificationPanel />}
-            {currentView === INGESTION_VIEWS.BOOKMARKLET && <BookmarkletPanel />}
+            {currentView === INGESTION_VIEWS.GENERATING_DEMO_DATA && <GeneratingDemoDataPanel />}
             {currentView === INGESTION_VIEWS.CHOOSE_THIRD_PARTY && <ThirdPartyPanel />}
+            {currentView === INGESTION_VIEWS.NO_DEMO_INGESTION && <NoDemoIngestionPanel />}
```

Comment:
FYI locally if the demo project wouldn't have the `onboarding-v2-experiment` flag so you would always see the `v1` ingestion and never get to this view.

I don't think that would happen in production though.

Reply:
Hm. A certain person should always keep their feature flag values, correct?

Reply:
We can do that. Yes it should always show the same feature flag version.